packages/kbn-optimizer/limits.yml

```diff
@@ -115,7 +115,7 @@ pageLoadAssetSize:
   presentationUtil: 58834
   profiling: 36694
   remoteClusters: 51327
-  reporting: 57003
+  reporting: 78653
```

Comment:
This increase might not be needed after taking `lens` out of the Reporting dependencies, and no longer using `import url from 'url'` in Reporting.

Reply:
wooo I'll try and reduce it 👍🏻

Reply:
10476c9 for changes